pandas/core/frame.py

```diff
@@ -109,10 +109,9 @@
 _shared_doc_kwargs = dict(
     axes='index, columns', klass='DataFrame',
     axes_single_arg="{0 or 'index', 1 or 'columns'}",
-    axis="""
-    axis : {0 or 'index', 1 or 'columns'}, default 0
-        - 0 or 'index': apply function to each column.
-        - 1 or 'columns': apply function to each row.""",
+    axis="""axis : {0 or 'index', 1 or 'columns'}, default 0
+        If 0 or 'index': apply function to each column.
+        If 1 or 'columns': apply function to each row.""",
     optional_by="""
```

Comment:
``axis`` doesn't line up with the previous line in master, so generated doc string doesn't show axis etc, see [here in master](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.Series.transform.html).

The above change lines it properly up, so it shows up as expected.